exporters/otlp/trace/build.gradle.kts

```diff
@@ -53,8 +27,50 @@ dependencies {
   jmhRuntimeOnly("io.grpc:grpc-netty")
 }
 
+testing {
+  suites {
+    val testGrpcNetty by registering(JvmTestSuite::class) {
+      dependencies {
+        implementation(project(":exporters:otlp:testing-internal"))
+
+        implementation("io.grpc:grpc-netty")
+        implementation("io.grpc:grpc-stub")
+      }
+    }
+    val testGrpcNettyShaded by registering(JvmTestSuite::class) {
+      dependencies {
+        implementation(project(":exporters:otlp:testing-internal"))
+
+        implementation("io.grpc:grpc-netty-shaded")
+        implementation("io.grpc:grpc-stub")
+      }
+    }
+    val testGrpcOkhttp by registering(JvmTestSuite::class) {
+      dependencies {
+        implementation(project(":exporters:otlp:testing-internal"))
+
+        implementation("io.grpc:grpc-okhttp")
+        implementation("io.grpc:grpc-stub")
+      }
+    }
+    val testSpanPipeline by registering(JvmTestSuite::class) {
```

Comment:
I think if we do not use the result of registering, then the `by registering` syntax looks odd. Isn't there some `register(JvmTestSuide::class)` API that could be used instead?

Reply:
`register` with a string is the alternative. I find it more consistent to just always use this form even when not using the variable, here we happen to not have any need for variable form but if we happened to mix the two patterns it is confusing I think.